examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -470,15 +495,29 @@ func extractNonPointerInterfaces(comments []string) (bool, error) {
 	if len(values) == 0 {
 		return false, nil
 	}
-	result := values[0] == "true"
+	result := values[0] == "true" || values[0] == ""
```

Comment:
We do a poor job overall of normalizing and validating these tags.  Would be nice to not have to consider "" everywhere.